Framework/API/src/Algorithm.cpp

```diff
@@ -584,6 +591,16 @@ bool Algorithm::execute() {
       setExecuted(true);
 
       // Log that execution has completed.
+      getLogger().debug() << "Time to validate properties: " +
```

Comment:
I'd meant something more like:
```
std::string dbgMsg = "Time to validate properties: ";
dbgMsg.append(std::to_string(timingPropertyValidation)).append(" seconds\n");

+ similar append lines for other info...

# and finally send it to the debug log stream in one shot. The stream api flushes each time it encouters a "\n" so
# this will just send the whole lot out once.
getLogger().debug(dbgMsg); 
```